tests/test_analysis.py

```diff
@@ -1421,3 +1509,9 @@ def test_assess_disease_prioritisation_threshold_meets_cutoff(self):
                 }
             },
         )
+        self.assertEqual(
+            self.binary_classification_stats,
+            BinaryClassificationStats(
+                true_positives=1, true_negatives=3, false_positives=0, false_negatives=0
```

Comment:
Are these RankStats correct?
```
RankStats(
    top=1,
    top3=1,
    top5=1,
    top10=1,
    found=1,
    total=1,
    reciprocal_ranks=[1.0],
    relevant_result_ranks=[[1]],
),

true_positives=1, true_negatives=3, false_positives=0, false_negatives=0
```

This looks like found and total should be 4, not 1?

Reply:
This is correct. Found and total should be 1 as the found and total point to the number of relevant entities found. But the TP, TN, FP, FN points to all the results, relevant or not.